rails-install-ubuntu.sh

```diff
@@ -2,50 +2,69 @@
 
 set -e
 
+echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
+sudo add-apt-repository ppa:chris-lea/node.js
+
+echo "Adding deb. repository for PostgreSQL"
```

Comment:
I'm not running Ubuntu myself, and I didn't find an explanation in the commit messages. Did the Postgres setup below not work? What is the reason for this change?